packages/frontend/src/components/profile_pic.tsx

```diff
@@ -0,0 +1,78 @@
+import type { Sport } from "@esp-group-one/types";
+import { useState } from "react";
+import { useEffect } from "react";
+import { Tag } from "./tags";
+
+interface Info {
+  sportName: Sport;
+  selected: boolean;
+  abilityLevel: string;
+  onClick: () => void;
+}
+
+interface SportInfo {
+  sports: Info[];
+}
+
+const getAbilityLevel = (sports: Info[]): string => {
+  const selectedSport = sports.find((sport) => sport.selected);
+
+  return selectedSport ? selectedSport.abilityLevel : "Beginner";
+};
+
+function getAbilityColour(abilityLevel: string) {
+  switch (abilityLevel) {
+    case "Beginner":
+      return "bg-beginner";
+    case "Intermediate":
+      return "bg-intermediate";
+    case "Advanced":
+      return "bg-advanced";
+  }
+}
+
+export function ProfilePic({
+  sports: initialSports,
```

Comment:
This seems a lot of information to pass in...

Could we not just pass in an array of `SportInfo` (the one from types packages), which then build the rest of the information.

Also while thinking about it, it might be useful to also pass in an `onChange: (sport: Sport) => void` function